### PR TITLE
Feature: validators list

### PR DESCRIPTION
**Pull request by KrzysztofLabus-RDX**
**Note** this lovely `object stake = new { Xxx = "zzzz" }; // TODO fix me!` is essentially a placeholder for Validator's stake we'll develop as separate PR

### CODE REVIEW

File: src/RadixDlt.NetworkGateway.GatewayApi/Validators/StateValidatorsListRequestValidator.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+namespace RadixDlt.NetworkGateway.GatewayApi.Validators;
+
+internal class StateValidatorsListRequestValidator : AbstractValidator<GatewayModel.StateValidatorsListRequest>
+{
+    public StateValidatorsListRequestValidator(LedgerStateSelectorValidator ledgerStateSelectorValidator)
+    {
+        RuleFor(x => x.AtLedgerState)
+            .SetValidator(ledgerStateSelectorValidator);
+
+        RuleFor(x => x.Cursor);
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.GatewayApi/Validators/StateValidatorsListRequestValidator.cs:77]: any reason why we leave it without any explicit validators?

File: src/RadixDlt.NetworkGateway.PostgresIntegration/Services/EntityStateQuerier.cs [hunk 2]
```diff
@@ -385,6 +386,77 @@ LIMIT 1
...
+            .Take(ValidatorsLimit)
+            .Select(v =>
+            {
+                var address = v.BuildHrpGlobalAddress(_networkConfigurationProvider.GetHrpDefinition());
+
+                GatewayModel.ValidatorCollectionItemActiveInEpoch? activeInEpoch = null;
+
+                if (activeSetById.TryGetValue(v.Id, out var vk))
+                {
+                    object stake = new { Xxx = "zzzz" }; // TODO fix me!
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.PostgresIntegration/Services/EntityStateQuerier.cs:444]: not sure what's that?

**Review by PawelPawelec-RDX — COMMENTED** [review]
Don't we need any tests for created logic?

File: src/RadixDlt.NetworkGateway.PostgresIntegration/Services/EntityStateQuerier.cs (hunk 2 above)
Comment (KrzysztofLabus-RDX) [reply src/RadixDlt.NetworkGateway.PostgresIntegration/Services/EntityStateQuerier.cs:444]: > Note this lovely `object stake = new { Xxx = "zzzz" }; // TODO fix me!` is essentially a placeholder for Validator's stake we'll develop as separate PR

File: src/RadixDlt.NetworkGateway.GatewayApi/Validators/StateValidatorsListRequestValidator.cs (hunk 1 above)
Comment (KrzysztofLabus-RDX) [reply src/RadixDlt.NetworkGateway.GatewayApi/Validators/StateValidatorsListRequestValidator.cs]: Fixed

**Review by PawelPawelec-RDX — APPROVED** [review]

**Pull request merged**